internal/scorecard/alpha/formatting.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alpha
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/operator-framework/operator-sdk/pkg/apis/scorecard/v1alpha2"
+	"k8s.io/client-go/kubernetes"
+)
+
+// getTestResults fetches the test pod logs and converts it into
+// ScorecardOutput format
+func getTestResults(client kubernetes.Interface, tests []ScorecardTest) (output v1alpha2.ScorecardOutput) {
+	output.Results = make([]v1alpha2.ScorecardTestResult, 0)
+	for i := 0; i < len(tests); i++ {
+		p := tests[i].TestPod
+		logBytes, err := getPodLog(client, p)
+		if err != nil {
+			fmt.Printf("error getting pod log %s\n", err.Error())
+		} else {
+			// marshal pod log into ScorecardTestResult
+			var sc v1alpha2.ScorecardTestResult
+			err := json.Unmarshal(logBytes, &sc)
+			if err != nil {
+				fmt.Printf("error unmarshalling test result %s\n", err.Error())
```

Comment:
Should not it is a fatal log error?

Reply:
I updated this code to now pass back an error so that cobra can print the error correctly, good catch.